src/assets/guide-markdown/data-requirements.md

```diff
@@ -25,45 +25,47 @@ or
 
 The Human Cell Atlas Data Coordination Platform accepts datasets featuring both data access types, with an important note on the data and metadata files required for each type:
 
-## Open access accepted data and metadata:
+## Open access datasets - accepted data and metadata:
 
-Raw sequencing files (fastq, bam)
+* Raw sequencing files (fastq, bam)
 
-Donor and sample metadata
+* Donor and sample metadata
 
-Analysis files (raw/normalised matrices; cell type annotations; spatial embeddings)
+* Analysis files (raw/normalised matrices; cell type annotations; spatial embeddings)
 
 
-## Controlled access accepted data and metadata:
+## Controlled access datasets - accepted data and metadata:
 
-Sample metadata (no donor-identifiable metadata)
+* Sample metadata (no donor-identifiable metadata)
 
-Analysis files (raw/normalised matrices; cell type annotations; spatial embeddings)
+* Analysis files (raw/normalised matrices; cell type annotations; spatial embeddings)
 
-# Data eligibility
+# Data sources
 
 There are three main sources of data considered eligible for the Human Cell Atlas:
 
-## Data referenced for the Human Cell Atlas Bionetwork Atlases
+* **Data referenced for the Human Cell Atlas Bionetwork Atlases**
```

Comment:
Using bullet points for these makes the related content look strange, I think a numbered list with an indent looks good
```suggestion
1. **Data referenced for the Human Cell Atlas Bionetwork Atlases**
```